package.json

```diff
@@ -9,13 +9,14 @@
     "type-defintions/dom-event.d.ts"
   ],
   "scripts": {
-    "build-dist": "mkdir -p dist && rollup src/dom-event.js | babel --presets es2015 --plugins transform-es2015-modules-umd --module-id '@most/dom-event' -o dist/dom-event.js",
-    "build": "npm run build-dist && uglifyjs dist/dom-event.js -o dist/dom-event.min.js",
+    "build": "npm run build-dist && uglifyjs dist/mostPackage.js -o dist/mostPackage.min.js",
+    "build-dist": "mkdir -p dist && rollup -c --name '@most/mostPackage' | umd-name-transform -o dist/mostPackage.js",
```

Comment:
`rollup -c` assumes that there is a file `rollup.config.js` is it not neeed here?